lib/jekyll/command.rb

```diff
@@ -55,6 +55,7 @@ def add_build_options(c)
         c.option "limit_posts", "--limit_posts MAX_POSTS", Integer,
           "Limits the number of posts to parse and publish"
         c.option "watch", "-w", "--[no-]watch", "Watch for changes and rebuild"
+        c.option "baseurl", "-b", "--baseurl", "Serve the website from the given base URL"
```

Comment:
This option takes an argument so the third argument here must be `--baseurl URL` or something so OptionsParser does the Right Thing. Without this, it will only return `true` (when present) or `false` (when omitted).